CHANGELOG.rst

```diff
@@ -1,6 +1,19 @@
 Change Log
 ==========
 
+v3.6.0 (UNRELEASED)
+-------------------
+
+In this release we deprecated feature that were announced to be deprecated some time ago. We deprecated all compatibility function with 'KE-chain 2'.
+
+* :+1: Added `editable` argument to the `add_attachmentviewer_widget` method of the WidgetsManager.
+* :+1: Added `Alignment` enum class, leaving `NavigationBarAlignment` as wrapper for backwards compatibility.
+
+BIX - Backward Incompatbile Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We deprecated ... TODO: fill this
```

Comment:
We need to fill in some documentation here what we actually deprecated.

Reply:
Done, see changelog.